packages/components/src/Heading.tsx

```diff
@@ -47,10 +47,19 @@ export const BlockHeading = styled.h3`
 
 BlockHeading.defaultProps = headingDefaultProps;
 
-export const FormHeading = styled.h3`
+export const ManagerHeading = styled.h3`
   font-family: ${props => props.theme.sansSerifFont};
-  font-weight: 400;
-  font-size: 24px;
+  font-weight: 600;
+  font-size: 18px;
+`;
+
+ManagerHeading.defaultProps = headingDefaultProps;
+
+// @TODO weight 500 for semi-bold works with our font stack on Mac but not on Linux, should check Windows cause this heading looks dumb in 400 weight which browser uses if semi-bold not available.
```

Comment:
Oh yeah good call, will move it there, especially since we might have semi-bold elsewhere and we should decide whether to update font stack accordingly, include a fallback font, not use it, or whatever.